lib/enkf/tests/row_scaling.cpp

```diff
@@ -118,9 +118,11 @@ void test_multiply() {
   {
     row_scaling_type * row_scaling = row_scaling_alloc();
     matrix_type * A = matrix_alloc_copy(A0);
+    std::vector<float> row_data(data_size);
     for (int row = 0; row < data_size; row++)
-      row_scaling_iset(row_scaling, row, 0);
-    row_scaling_multiply(row_scaling, A, X0);
```

Comment:
I don't understand my the multiply call is removed

Reply:
Because the same operation is now performed with the `row_scaling_assign_float()` function call.

Reply:
I am not seeing how these are the .  Would love to get an explanation :)

Reply:
I am sorry you are right; the multiply operation should be moved after the new `row_scaling_assign_float()`function call but not completely removed.

My mistake. Fixed.